### ISSUE TITLE
I don't deserialize java 8 date/time type `java.time.Instant`

### ISSUE THREAD

**Issue by benjessie25** [FasterXML/jackson-modules-java8#302]

**Comment by benjessie25** [comment]
com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Java 8 date/time type `java.time.Instant` not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" to enable handling

**Comment by cowtowncoder** [comment]
Yes, if you have that type somewhere in your class definitions, you need to either add the module, or add something else to handle it (custom (de)serializer), or mark as "ignored type".
Something like:

```
        mapper.configOverride(Instant.class).setIsIgnoredType(true);
```

which would then ignore all properties with declared type of `Instant`.
